src/xrCore/ModuleLookup.cpp

```diff
@@ -37,7 +37,7 @@ void* ModuleHandle::Open(pcstr moduleName)
 #ifdef WINDOWS
```

Comment:
Get rid of this ugly ifdef, please - create and call platform-dependent functions, please.

Reply:
We have decided that it's better to split cpp to Windows and Linux versions (`ModuleLookupWindows.cpp` and `ModuleLookupLinux.cpp`) but that's not related to this PR and this should be done later.